skore/tests/unit/sklearn/estimator/test_estimator.py

```diff
@@ -1341,3 +1342,97 @@ def test_estimator_report_average_return_float(binary_classification_data):
     for metric_name in ("precision", "recall", "roc_auc"):
         result = getattr(report.metrics, metric_name)(average="macro")
         assert isinstance(result, float)
+
+
+def test_estimator_report_metric_with_neg_metrics():
+    """Check that scikit-learn metrics with 'neg_' prefix are handled correctly."""
+
+    X_train, X_test, y_train, y_test = train_test_split(
+        *load_breast_cancer(return_X_y=True), random_state=0
+    )
+    classifier = LogisticRegression(max_iter=10_000)
+
+    report = EstimatorReport(
+        classifier,
+        X_train=X_train,
+        y_train=y_train,
+        X_test=X_test,
+        y_test=y_test,
+    )
+
```

Comment:
You should be able to use the `binary_classification_data` fixture:
```suggestion
def test_estimator_report_metric_with_neg_metrics(binary_classification_data):
    """Check that scikit-learn metrics with 'neg_' prefix are handled correctly."""

    classifier, X_test, y_test = binary_classification_data

    report = EstimatorReport(
        classifier,
        X_test=X_test,
        y_test=y_test,
    )
```
You might need to use a different metric than log loss since the estimator is a RandomForestClassifier.

Same for the other test.